app/project/[id]/components/project-task-files.tsx

```diff
@@ -25,14 +29,43 @@ const filesColumns: Tan.ColumnDef<DesignFile>[] = [
     cell: ({ row }) => <pre>{row.getValue("filename")}</pre>,
   },
   {
-    accessorKey: "size",
+    accessorKey: "uploadedat",
+    header: () => <div>Uploaded</div>,
+    cell: ({ row }) => {
+      return (
+        <pre>
+          {format(
+            new Date(row.getValue("uploadedat")),
+            "MM/dd/yyyy 'at' h:mm a",
+          )}
+        </pre>
+      );
+    },
+  },
+  {
+    accessorKey: "uploader",
+    header: () => <div>Uploader</div>,
+    cell: ({ row }) => (
+      <div className="flex overflow-hidden w-12 items-center justify-center">
+        <UserAvatar user={row.getValue("uploader")} />
+      </div>
+    ),
+  },
+  {
+    accessorKey: "filesize",
     header: () => <div>Size</div>,
-    cell: ({ row }) => <pre>{row.getValue("size")}</pre>,
+    cell: ({ row }) => <pre>{getFileSize(row.getValue("filesize"))}</pre>,
   },
   {
     accessorKey: "taskid",
     header: () => <div>Task Id</div>,
-    cell: ({ row }) => <pre>{row.getValue("taskid")}</pre>,
+    cell: ({ row, projectid }: any) => (
+      <Button asChild variant="outline" className="h-8 w-8 p-0">
+        <Link href={`/project/${projectid}/task/${row.getValue("taskid")}`}>
```

Comment:
confusingly this should actually be specid
so each taskid is unique but each project can have similar specids e.g. /project/xxx/task/100 will always be "Title Search" etc..

Reply:
Also if there is a comment linked with this file, i'd like to link to that comment e.g.
https://haru2-kappa.vercel.app/project/365/task/100#comment-137